packages/wallets/src/AppContent.scss

```diff
@@ -14,7 +14,7 @@
     }
 }
 
-.wallets-modal-responsive-root {
+.wallets-modal-show-header-root {
     position: absolute;
     z-index: 1;
 }
```

Comment:
Update, to hide the Deriv.app header, from now you can:
1. Set `shouldHideDerivAppHeader` in `show` option from `useModal` hook. 
   - If set to true, the modal provider will mount the modal in the root container at Deriv.app level for responsive (where the modal will occupy full-screen and hide the Deriv.app header)
   - Otherwise it will default to mounting it within our Wallets level (where the Deriv.app header is not covered)
3. Set `shouldHideDerivAppHeader` in `ModalStepWrapper` prop. 
   - If set to true, it will do the similar action as setting `showHideDerivAppHeader` to `true` in `show` option